nvdaHelper/remote/winword.cpp

```diff
@@ -304,6 +304,12 @@ int generateHeadingXML(IDispatch* pDispatchParagraph, IDispatch* pDispatchParagr
 	int headingLevel=getHeadingLevelFromParagraph(pDispatchParagraph);
 	if(!headingLevel) return 0;
 	XMLStream<<L"<control role=\"heading\" level=\""<<headingLevel<<L"\" ";
+	// Expose the collapsed state of the heading
+	bool isCollapsed=false;
+	_com_dispatch_raw_propget(pDispatchParagraph, wdDISPID_PARAGRAPH_COLLAPSED_STATE, VT_BOOL, &isCollapsed);
+	if(isCollapsed) {
+		XMLStream<<L"collapsedState=\"true\" ";
+	}
```

Comment:
Should we explicitly set `collapsedState` to `false` as well, or is the implicit value fine?
```suggestion
	XMLStream<<L"collapsedState=\""<<(isCollapsed ? L"true" : L"false")<<L"\" ";
```